install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -143,8 +143,10 @@ data:
           # Address of the Zipkin collector
         {{- if .Values.global.tracer.zipkin.address }}
           address: {{ .Values.global.tracer.zipkin.address }}
+        {{- else if .Values.global.istioNamespace }}
```

Comment:
Why not change `.Values.global.istioNamespace` to `{{ .Release.Namespace }}`?
Helm/Tiller are supposed to manage resource in same namespace.

Reply:
This is to construct an arg for tracing address, but not a resource. But anyway if it is general practice to use release.namespace, I think it also makes sense to use it as default domain here.